src/components/BounceRulesActivity/index.scss

```diff
@@ -0,0 +1,69 @@
+.container {
+  h1 {
+    margin-bottom: 0;
+  }
+}
+
+table {
```

Comment:
Same here the styles for all table elements will override/cascade down to other table elements in general if we're not careful and will affect the overall css in different ways. let's add a class to the table to be less specific and lock it down to only the table we are trying to affect